docs/_source/troubleshooting-faqs/code-questions.rst

```diff
@@ -11,27 +11,88 @@ Frequently Asked Questions
+        :alt: Memory usage of two large population synthesis runs.
+        :width: 700px
+    
+    The memory usage of the 8000 :code:`dump_rate` run is stable at around 6GB, while the 10.000 :code:`dump_rate` run is stable at around 6.8GB.
+
+
+4. **What should the walltime and job array size be for my population synthesis run?**
+
+    The :code:`walltime`` and job array size are dependent on the number of binaries you want to simulate and the memory usage of the simulation.
+    The job array size should be set such that the number of binaries per job is at least 1000, since there's a minimum overhead per job due to loading the grids.
+    
+    The :code:`walltime` depends on the number of binaries per job, where each binary takes about 1-2 seconds to run.
+    For example, with 100.000 binaries split over 100 jobs (per metallicity), means that every job runs 1.000 binaries. This will take around 33 minutes per job. So a :code:`walltime` of `00:45:00` is reasonable.
+
+    The balance between :code:`walltime` and the size of the job array is important.
+    If the :code:`walltime` is too long, it might be worth increasing the job array size to decrease the time per job and allowing the population synthesis to finish faster. 
+    But if the :code:`walltime` is too short, the job array size should be decreased, since each job has an initial overhead that is not dependent on the number of binaries in the job.
+
+    .. note::
+        The processing time increases if you make the `dump_rate` too low due to many I/O operations.
+
+5. **I am unable to open HDF5 files created by POSYDON. What should I do?**
+    
+    If you're on a Mac, there might be an issue with the HDF5 installation.
+    Make sure you have the `hdf5` and `pytables` packages installed through conda in your environment with `conda install hdf5 pytables` before running POSYDON!
+    Although they are dependencies of POSYDON, sometimes they're not installed correctly on Mac.
 
 6. **Are there any examples or tutorials available?**
     - Answer: Yes, you can check our :ref:`roadmap <roadmap>` for tutorials related to different POSYDON components, including population synthesis, creating core datasets, and running your own MESA grids with POSYDON.
 
 7. **Can I run POSYDON on an HPC facility?**
-    - Answer: Absolutely! Ensure you've installed ``mpi4py`` as outlined in :ref:`our installation guide <installation-guide>`. Refer to `our HPC guide <../tutorials-examples/population-synthesis/pop_syn.ipynb>`_ for detailed instructions on running POSYDON in an HPC environment.
+    - Answer: Absolutely! Refer to `our HPC guide <../tutorials-examples/population-synthesis/pop_syn.ipynb>`_ for detailed instructions on running POSYDON in an HPC environment.
 
 8. **How can I stay updated with the latest features and updates?**
     - Answer: You can regularly visit our `official website <https://posydon.org>`_ for news and updates. Also, consider subscribing to our mailing list.
```

Comment:
Here we should add instructions, how to subscribe to the mailing list, or link to them.